src/Servers/Kestrel/Core/src/Internal/Http/HttpHeaders.cs

```diff
@@ -564,6 +564,14 @@ public static TransferCoding GetFinalTransferCoding(StringValues transferEncodin
                             offset += sizeof(uint) / 2;
                             transferEncodingOptions = TransferCoding.Chunked;
                         }
+                        else
+                        {
+                            transferEncodingOptions = TransferCoding.Other;
```

Comment:
I wonder if we could start of `transferEncodingOptions` uninitialized inside the loop to make sure each iteration explicitly sets a value rather than accidently leave it as `TransferCoding.None` when we shouldn't. Or wrongly carry over the value from a previous duplicate Trasnfer-Encoding header. Or wrongly carry over the value from before the last comma.

Reply:
Do you see any other cases still missed here?

Refactoring this method is more risk than necessary for 6.0 or 7.0. Do you see value in doing this for 8?

Reply:
Given that we want to treat "chunked," the same as "chunked", I think it's fine. I understand not wanting to change it too much, it just struck me how easy this bug was to miss.